ideas/121-swarm-compensation.md

```diff
@@ -0,0 +1,72 @@
+    Created: 2018-04-02
+    Related: `120-swarm-process`
+
+## Summary
+Ultimately we would like Status to become a public good, owned, developed and ran by it's very own users. In order to achieve that we need to open up our processes for contributions from "the outside" and establish ways for users to propose & incentivise projects they see as important.
+
+Swarms are our current attempt at enabling this. They are ephemeral teams forming to achieve a specified goal and disbanding once achieved or failure becomes apparent. Their defined lifetime and -cycle encourage constant re-evaluation &. evolution.
+
+Compensating contributors for their work is an important component of this and where `120-swarm-process` focuses on establishing a baseline of processes that are open to everyone this idea will focus on experimentally devising compensation mechanisms. 
+
+## Swarm Participants
+
+- Lead Contributor: @martinklepsch
+- Contributor ('payment'): ??? 
+- Contributor: @oskarth
+- Evaluation ('compensation criteria'): Voting mechanism (Polly survey)
+
+## Product Overview
+
+This swarm will define **four** experimental compensation mechanisms and recruit upcoming swarms to participate in this experiment.
+
+The compensation mechanisms should be judged according to the following criteria:
+
+- **Fairness** — if we're not fair we will fail to retain contributors
+- **Ease of use** — we want to ship a new kind of platform, not get bogged down in "paperwork"
+- **More attractive than salary** — as outlined in `120-swarm-process` there are two types of swarms:
+	- **`Type-1`** is the ideal, defined, ran and compensated by the community
+	- **`Type-2`** is the currently predominant form. Defined, ran and compensated by employees of Status, the company.
+	- While coming up with a nice model for `Type-1` swarms we also would like to encourage every Status employee to participate in similar ways as community contributors.
+
+## Checkpoints
+
+#### Checkpoint 1 — `2018-04-06`
+
+- Define at least two compensation models
+- Get buy-in from major SNT holder to fund experiments
+- Talk to #peopleops how we can admin/structure this
+- Reach out to upcoming swarms and convince them to participate
+- Broadcast intention behind these experiments into community and encourage submission of ideas. (Medium post?)
+
+#### Checkpoint 2 — `2018-04-20`
+
+- Define two more compensation models
+- Again, reach out and convince upcoming swarms
+	- If possible consider a community submitted idea/swarm
+
+#### Checkpoint 3 — `2018-05-31`
+
+- Summarize learnings and archive them in ideas repository.
+- Write up what these learnings mean for the OpenBounty project specifically and how they should affect the development of the product.
+
+## Exit Criteria
+
+Conduct four experiments and evaluate them based on the criteria outlined above. Give recommendation on what kind of compensation mechanism should be enabled by Status OpenBounty.
+
+## Metrics Affected
```

Comment:
Success metrics? Can we phrase this as increasing transaction volume by `ballpark figure` and MAC by `ballpark`?

Reply:
Renamed this to "Metrics Affected" because specific numbers seemed like random shots in the dark for no reason. 

MAC might not be affected by this at all assuming most contributors are coming from "core" (i.e. Status, the company).